### PR TITLE
Disable Facebook link tracking

### LINKED ISSUE #224: Block Facebook link tracking
It would be cool if Facebook link tracking was possible to disable. Currently e.g. clicking on a sent https://google.com really opens https://l.messenger.com/l.php?u=https%3A%2F%2Fgoogle.com%2F&h=A_VERY_LONG_TRACKING_ID

### PR DESCRIPTION
**Pull request by kevva**
Fixes #224.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -222,6 +223,10 @@ app.on('ready', () => {
 				e.newGuest = new electron.BrowserWindow(options);
 			}
 		} else {
+			if (url.includes('://l.messenger.com/l.php')) {
```
Comment (sindresorhus) [inline index.js]: Can't you use `if (url.startWith('https://l.messenger.com/l.php')) {`? I'm pretty sure Facebook only uses HTTPS.

File: index.js [hunk 2]
```diff
@@ -222,6 +223,10 @@ app.on('ready', () => {
...
+				url = urlMod.parse(url, true).query.u;
```
Comment (sindresorhus) [inline index.js]: Can you use the ne URL API: https://nodejs.org/api/url.html#url_the_whatwg_url_api

**Comment by sindresorhus** [comment]
Can you add this feature to the readme?

File: index.js (hunk 2 above)
Comment (kevva) [inline index.js]: Good idea. Fixed!

**Comment by sindresorhus** [comment]
Thanks Kevin. As a bonus this also makes the links open much faster, since there's no longer a slow redirect.

**Comment by kevva** [comment]
🎉

**Pull request merged**